frontend/src/store/modules/compare/actions.ts

```diff
@@ -81,20 +81,20 @@ export default {
         })
     },
 
-    async setComparedRepos(context:any, payload:any) {
-        return new Promise((resolve:any, reject:any)=>{
-            setTimeout(()=>{
-                let promises:any[] = [];
-                for(let repo of payload) {
+    async setComparedRepos(context: any, payload: any) {
```

Comment:
Looking over the difference between `let` and `const`, I think `const` makes sense for the repo and repogroup stuff ... For compared repos, however, it seems like we want to be able to change these bindings if I am reading the documentation and understand correctly (who knows?)

Reply:
@bing0n3 and @gabe-heim ... let us know what you think.